src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -1,107 +1,38 @@
-
-    <!-- If the stage 0 SDK doesn't have KnownWorkloadManifests.txt, then write a default version -->
-    <WriteLinesToFile File="$(SdkOutputDirectory)\KnownWorkloadManifests.txt"
-                      Lines="microsoft.net.sdk.android;microsoft.net.sdk.aspire;microsoft.net.sdk.ios;microsoft.net.sdk.maccatalyst;microsoft.net.sdk.macos;microsoft.net.sdk.maui;microsoft.net.sdk.tvos;microsoft.net.workload.mono.toolchain.current;microsoft.net.workload.emscripten.current;microsoft.net.workload.emscripten.net6;microsoft.net.workload.emscripten.net7;microsoft.net.workload.emscripten.net8;microsoft.net.workload.emscripten.net9;microsoft.net.workload.mono.toolchain.net6;microsoft.net.workload.mono.toolchain.net7;microsoft.net.workload.mono.toolchain.net8;microsoft.net.workload.mono.toolchain.net9"/>
-
+    <!-- Copy artifacts from the stage0 SDK folder.
+         Ignore the sdk, templates and host files for which only live built artifacts should be used. -->
     <ItemGroup>
-      <WasmWorkloads Include="microsoft.net.workload.mono.toolchain.current" Version="$(MonoWorkloadManifestVersion)/$(MonoWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.mono.toolchain.net6" Version="$(MonoWorkloadManifestVersion)/$(MonoWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.mono.toolchain.net7" Version="$(MonoWorkloadManifestVersion)/$(MonoWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.mono.toolchain.net8" Version="$(MonoWorkloadManifestVersion)/$(MonoWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.mono.toolchain.net9" Version="$(MonoWorkloadManifestVersion)/$(MonoWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.emscripten.current" Version="$(EmscriptenWorkloadManifestVersion)/$(EmscriptenWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.emscripten.net6" Version="$(EmscriptenWorkloadManifestVersion)/$(EmscriptenWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.emscripten.net7" Version="$(EmscriptenWorkloadManifestVersion)/$(EmscriptenWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.emscripten.net8" Version="$(EmscriptenWorkloadManifestVersion)/$(EmscriptenWorkloadFeatureBand)"/>
-      <WasmWorkloads Include="microsoft.net.workload.emscripten.net9" Version="$(EmscriptenWorkloadManifestVersion)/$(EmscriptenWorkloadFeatureBand)"/>
+      <_OverlaySDKFile Include="$(_DotNetHiveRoot)\**\*"
+                       Exclude="$(_DotNetHiveRoot)sdk\**\*;
+                                $(_DotNetHiveRoot)templates\**\*;
+                                $(_DotNetHiveRoot)host\**\*"/>
+      <OverlaySDKFile Include="@(_OverlaySDKFile->'$(TestHostFolder)%(RecursiveDir)%(Filename)%(Extension)')" Source="%(Identity)" />
```

Comment:
did you check that this only contains expected files?

Reply:
Yes, I looked at the Copy task in the binlog and the expected live and LKG artifacts are being used now.